src/Bridge/Symfony/Bundle/Resources/config/swagger.xml

```diff
@@ -18,19 +18,19 @@
             <argument>%api_platform.title%</argument>
```

Comment:
Should be injected in the `DocumentationAction` and passed to the constructor of the new `Documentation` object.